aerogear/mobile-security-service
# Added a Makefile to automate build and testing tasks

## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-8397

## What

Create a Makefile to automate many tasks such as:

* Installing dependencies
* Building images
* Testing
* Error checking and code inspection

Also added these commands to the README[#building--testing](https://github.com/craicoverflow/mobile-security-service-server/tree/AEROGEAR-8397#building--testing)

## Why

Many of these commands are cumbersome and need to be repeated often so it is easier to tie them into Make commands instead.

## How

Used [aerogear-app-metrics](https://github.com/aerogear/aerogear-app-metrics)'s [Makefile](https://github.com/aerogear/aerogear-app-metrics/blob/master/makefile) to get most of the commands that are similar.

Changed tasks to suit current project.

## Verification Steps
 
1. Ensure `make` is in your system
2. Run each of the commands listed in [#building--testing](https://github.com/craicoverflow/mobile-security-service-server/tree/AEROGEAR-8397#building--testing).
3. For each command, check if the result correlates with what the command description says.

## Checklist:


## Progress
